example/example2.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 from __future__ import print_function
+import six
```

Comment:
Whoops, for some reason I thought that `six` was part of the standard library.  It was being used only for the `six.u` function, since unicode literals are not available in 3.0-3.2.  To avoid the dependency, I now replaced it with a function giving the same functionality.